From 0bd97e8aef5208bc92c3e72049f7fea198a90bd1 Mon Sep 17 00:00:00 2001
From: Henrik Riomar <henrik.riomar@gmail.com>
Date: Mon, 8 Jan 2018 16:37:44 +0100
Subject: [PATCH] skip hooks on diskless install

We can not run hooks before musl and busybox is installed.

Use the new flag --initramfs-diskless-boot in order to skip hooks.
This flag also implies --initdb and the relevant --force flags for
initramfs diskless boot.
---
 initramfs-init.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/initramfs-init.in b/initramfs-init.in
index f3a3ee5..7c78c58 100755
--- a/initramfs-init.in
+++ b/initramfs-init.in
@@ -598,7 +598,7 @@ if [ "$KOPT_chart" = yes ]; then
 	pkgs="$pkgs acct"
 fi
 
-apkflags="--initdb --progress --force"
+apkflags="--initramfs-diskless-boot --progress"
 if [ -z "$ALPINE_REPO" ]; then
 	apkflags="$apkflags --no-network"
 else
-- 
2.15.0

